fontforge/python.c

```diff
@@ -8521,49 +8540,76 @@ return(NULL);
 Py_RETURN( self );
 }
 
-/* PostScript importing flags */
+static char *glyph_import_keywords[] = { "filename", "correctdir",
+    "simplify", "handle_clip", "handle_eraser", "scale", "accuracy",
+    "default_joinlimit", "usesystem", "asksystem", NULL };
+
+/* Legacy PostScript importing flags */
 static struct flaglist import_ps_flags[] = {
     { "toobigwarn", 0 },			/* Obsolete */
     { "removeoverlap", 0 },			/* Obsolete */
-    { "handle_eraser", sf_handle_eraser },
-    { "correctdir",  sf_correctdir },
+    { "handle_eraser", 1 },
```

Comment:
So these `psstrokeflags` aren't used any more? Can we remove them?

Reply:
Apparently so